src/main/java/com/scottlogic/TopicSort.java

```diff
@@ -0,0 +1,67 @@
+package com.scottlogic;
+
+import com.scottlogic.sorts.KeywordSearch;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class TopicSort {
+    public TreeMap<String, List<UserPost>> topicMap = new TreeMap<>();
+    List<UserPost> finalList = new ArrayList<>();
+    KeywordSearch sortByFrequencyOfTopic = new KeywordSearch();
```

Comment:
These two can be made private. It might also be a bit clearer to align the name of your class and variable for the KeywordSearch.